Bottom: 92e36281792aaecfa61ff06ccdfe90899711c66c
Top:    299a3edee6777df067c58ea99d2fa1da6cc284b1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 19:27:10 +0100

Refresh of doc-rat-lines-are-drawn-differ

---

diff --git a/doc/pcb.texi b/doc/pcb.texi
index 17f5051..cb09c00 100644
--- a/doc/pcb.texi
+++ b/doc/pcb.texi
@@ -2194,7 +2194,7 @@ If the layout is already completely wired, nothing will be added, and you will
 get a message that the wiring is complete.
 
 Rat-lines are lines having the special property that they only connect to pins and
-pads at their end points.  Rat-lines may be drawn different to other lines
+pads at their end points.  Rat-lines may be drawn differently to other lines
 to make them easier to identify since they have special behavior and cannot
 remain in a completed layout.
 Rat-lines are added in the minimum length straight-line tree pattern
